common/http_test.go

```diff
@@ -0,0 +1,169 @@
+package common_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"sync"
+	"testing"
+
+	"github.com/tendermint/tmlibs/common"
+)
+
+func TestWriteSuccess(t *testing.T) {
+	w := httptest.NewRecorder()
+	common.WriteSuccess(w, "foo")
+	got, want := w.Code, 200
+	if got != want {
```

Comment:
We have decided to use testify for all test cases. Can you port this... And the below checks

assert.Equal(want,got) instead of the if. 

Use require instead of assert if you want t.Fatal